src/hclib-deque.c

```diff
@@ -39,32 +39,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hclib-internal.h"
 #include "hclib-atomics.h"
 
-void deque_init(deque_t *deq, void *init_value) {
+#if 0 // UNUSED
```

Comment:
Absolutely. I would be more than happy to see all of those CUDA and UPC++ blocks deleted and that logic removed from the configure scripts. I've already done this in my branch (and replaced them with my own support that doesn't rely on preprocessor macros).